apps/src/templates/levelSummary/FreeResponseResponses.jsx

```diff
@@ -44,26 +66,26 @@ const FreeResponseResponses = ({responses, showStudentNames, eventData}) => {
           )}
         >
           <p>{response.text}</p>
-          <ResponseMenuDropdown
-            response={response}
-            hideResponse={userId => {
-              analyticsReporter.sendEvent(
-                EVENTS.CFU_RESPONSE_HIDDEN,
-                eventData,
-                PLATFORMS.BOTH
-              );
-              setHiddenResponses(prevHidden => [...prevHidden, userId]);
+          <ActionDropdown
+            name="free-response"
```

Comment:
how is this used

Reply:
this is part of the element ID of the div containing the dropdown trigger button. this would get translated to "free-response-dropdown" for the div's ID